client/package.json

```diff
@@ -1,5 +1,7 @@
 {
   "type": "commonjs",
   "main": "./index.js",
+  "react": "./react.js",
+  "jsx": "./jsx.js",
```

Comment:
What are these? This looks like it actually should be in a export config.

Reply:
Honestly, I don't know what was there... Fixed it.

Reply:
I was mostly just thinking about the imports that people have already. I'm pretty sure that you have everything wired up properly, but I just wanted to double check.